### PR TITLE
Add definitions for cookies v0.5.8

### PR DESCRIPTION
**Pull request by wangzishi**
Definitions for [cookies](https://github.com/pillarjs/cookies) v0.5.8

### CODE REVIEW

File: cookies/cookies.d.ts [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+declare module "cookies" {
+    import * as http from "http"
+
+    interface ICookies {
+        /**
+         * This extracts the cookie with the given name from the
+         * Cookie header in the request. If such a cookie exists,
+         * its value is returned. Otherwise, nothing is returned.
+         */
+        get(name: string): string;
```
Comment (panuhorsmalahti) [inline cookies/cookies.d.ts]: Is this ``get`` actually useful, since ``opts`` is an optional parameter below?

**Comment by panuhorsmalahti** [comment]
Looks good to me. :+1: for merging.

File: cookies/cookies.d.ts (hunk 1 above)
Comment (WangZishi) [inline cookies/cookies.d.ts]: You are right, I have fixed this.

**Comment by horiuchi** [comment]
@WangZishi Thanks mate!
@panuhorsmalahti Thank you for reviewing!

**Pull request merged**